std/assembly/index.d.ts

```diff
@@ -342,10 +344,15 @@ declare namespace i32 {
   export function div_s(left: i32, right: i32): i32;
   /** Computes the unsigned quotient of two 32-bit integers. */
   export function div_u(left: i32, right: i32): i32;
-  /** Return 1 two 32-bit inegers are equal to each other, 0 otherwise. */
+  /** Return 1 if two 32-bit integers are equal to each other, 0 otherwise. */
   export function eq(left: i32, right: i32): i32;
-  /** Return 0 two 32-bit inegers are equal to each other, 1 otherwise. */
+  /** Return 0 if two 32-bit integers are equal to each other, 1 otherwise. */
   export function ne(left: i32, right: i32): i32;
+  /** Compute the remaining between two 32-bit signed positive integers. */
+  export function rem_s(left: i32, right: i32): i32;
+  /** Compute the remaining between two 32-bit unsigned positive integers. */
+  export function rem_u(left: u32, right: u32): u32;
```

Comment:
```suggestion
  /** Computes the signed remainder of two 32-bit integers. */
  export function rem_s(left: i32, right: i32): i32;
  /** Computes the unsigned remainder of two 32-bit integers. */
  export function rem_u(left: u32, right: u32): u32;
```

Reply:
done in https://github.com/AssemblyScript/assemblyscript/pull/2306/commits/1130f0aac50bf058c79d91228484c4e872b53771